### ISSUE TITLE
Helm upgrade with install flag output is not same as upgrade or install output

### ISSUE THREAD

**Issue by hickeyma** [helm/helm#6172]
Output from `helm upgrade` with install flag `-i` is very terse one sentence output when it is doing an install. It does not mirror the output for an upgrade or an install.

Output of `helm version`:
$ helm version
version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"6bf5ef83c6f37bf12a59c108a094a8f3f93c83e1", GitTreeState:"clean", GoVersion:"go1.12.7"}

```console
$ helm install chrt-6076 chrt-6076/
NAME: chrt-6076
LAST DEPLOYED: 2019-08-07 12:49:01.541903521 +0100 IST m=+0.662510941
NAMESPACE: default
STATUS: deployed

NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods -l "app=chrt-6076,release=chrt-6076" -o jsonpath="{.items[0].metadata.name}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl port-forward $POD_NAME 8080:80

$ helm upgrade -i chrt-6076-1 chrt-6076/
Release "chrt-6076-1" does not exist. Installing it now.

$ helm upgrade -i chrt-6076-1 chrt-6076/
Release "chrt-6076-1" has been upgraded. Happy Helming!
NAME: chrt-6076-1
LAST DEPLOYED: 2019-08-07 12:50:14.19203183 +0100 IST
NAMESPACE: default
STATUS: deployed

NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods -l "app=chrt-6076,release=chrt-6076-1" -o jsonpath="{.items[0].metadata.name}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl port-forward $POD_NAME 8080:80
```

**Comment by waveywaves** [comment]
@hickeyma I have submitted a PR #6202 Do check :)

**Comment by bacongobbler** [comment]
fixed in #6202
